lessons/session_7/session_7_data_distribution_shifts.md

```diff
@@ -0,0 +1,40 @@
+---
+marp: true
+theme: uncover
+_class: invert
+paginate: true
+output: pdf_document
+---
+
+# ML Systems in Production
+# Session 7: Data Distribution Shifts
+Jesús A. Calderón
+February 2024
+
+---
+
+# Agenda
+
+**7.1. Data Distribution Shifts and Monitoring**
+	
++ ML System Failures
++ Data Distribution Shifts
++ Monitoring and Observability
+	
+**7.2 Continual Learning and Test in Production**
+	
+ + Continual Learning
+ + Stateless vs Stateful Training
+
+---
+
+# Agenda (cont.)
+
+**Assignment: Setting Up a Code Repository**
+
+ + Sign up for a GitHub Account
```

Comment:
This is a repetition, but I do understand that these are unfinished templates.

Reply:
Ok, removed.